### PR TITLE
[SD-3212] - Preserving desk during routing

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

**Comment by akintolga** [comment]
This PR is dependent on the pr superdesk/superdesk-core#16

**Comment by mdhaman** [comment]
:+1:

File: server/apps/rules/routing_rules.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 from superdesk.resource import Resource
 from superdesk.services import BaseService
 from superdesk.errors import SuperdeskApiError
-
+from eve.utils import config
 
 logger = logging.getLogger(__name__)
 STATE_ROUTED = 'routed'
```
Comment (mdhaman) [inline server/apps/rules/routing_rules.py]: This is already defined. Use CONTENT_STATE

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by marwoodandrew** [comment]
:+1:

**Pull request merged**